plugins/generator-1.16.5/forge-1.16.5/procedures/get_command_parameter_item.java.ftl

```diff
@@ -0,0 +1 @@
+(ItemArgument.getItem(arguments, "${field$param}").getItem())
```

Comment:
Actually, same applies for this, just with itemstack descriptor

Reply:
What can be applied here?

Reply:
BTW, why is this called `..._item.java.ftl` instead of `..._itemstack.java.ftl`?
```suggestion
/*@ItemStack*/(ItemArgument.getItem(arguments, "${field$param}").getItem())
```

Reply:
Because it gets and returns an item, so itemstack would not be the exact thing.

Reply:
But we work with item stacks...

Reply:
I am quite sure it returns itemstack

Reply:
Nah, even tests confirmed it doesn't match.

Reply:
They also confirm this change was not tested yet again :/

Reply:
They don't work with `/**Itemstack**/`, but the block works without it (e.g. adding the item to the player's inventory).

Reply:
> Hmm, I see now. But do we have any other procedure blocks doing this in MCreator, ie. returning item instead of itemstack?

Maybe, I would need to re-check. However, I can easily change it to get the default itemstack instance. It may simply add useless code, when a block get the item of an itemstack.

Reply:
I can say it is ok as it now for all cases, if we just find some other case where this is used and works so far :D

Reply:
I have only found `block_fluid_to_bucket.java.ftl` that is similar to this and it converts to itemstack, so it should be done here too

Reply:
Actually, the fix is already done I'm just working on remaking the needed templates.